src/daemon/nightlight.vala

```diff
@@ -74,9 +74,17 @@ namespace Budgie {
 
 			if (default_config_path != null && default_config_path != local_config_path) {
 				try {
-					// Copy the default configuration file to the local configuration file location
+					// Create the local folder to hold our configuration file
 					File dir = File.new_for_path(Path.get_dirname(local_config_path));
 					dir.make_directory_with_parents(null);
+				} catch (IOError.EXISTS exist_error) {
+					/* It's not an error to worry about if the folder exists */
+				} catch (Error e) {
+					warning("Failed to create local folder: %s\n", e.message);
```

Comment:
Do we need to exit here, if we failed to create the directory and it doesn't exist?

Reply:
Well, gammastep will still run, just not use any custom values set. The alternative is not having gammastep running at all... and then a user has to figure out why nothing appears to work when switching on/off.

Reply:
Hmm. So, our choices are either Gammastep runs with potentially unexpected settings, or it doesn't run at all. Neither is very great, at least not without somehow telling the user. Let's think about that later, though.